src/ui/Header.tsx

```diff
@@ -18,6 +22,14 @@ export default function Header(props: HeaderProps): React.ReactElement {
   const { headerLeft, state, navigator, manifest } = props;
   const linkColor = useColorModeValue('gray.700', 'gray.100', 'gray.700');
   const mainBgColor = useColorModeValue('ui.white', 'ui.black', 'ui.sepia');
+
+  // had to wrap this because I was having trouble adding custom styles to the NYPL Icon
```

Comment:
Nit: Code style wise, makes more sense to just go 
`// Add custom styles to the NYPL Icon`

(If you want to mention it to the PR reviewers, you can do it in the PR)

Reply:
got it, thanks!